nemo/collections/nlp/models/glue_benchmark/glue_benchmark_model.py

```diff
@@ -210,6 +221,19 @@ def multi_validation_epoch_end(self, outputs, dataloader_idx: int = 0):
         tensorboard_logs['val_loss'] = avg_loss
         return {'val_loss': avg_loss, 'log': tensorboard_logs}
 
+    def _setup_tokenizer(self, cfg: DictConfig):
+        tokenizer = get_tokenizer(
+            tokenizer_name=cfg.tokenizer_name,
+            data_file=cfg.data_file,
+            tokenizer_model=cfg.tokenizer_model,
+            sample_size=cfg.sample_size,
+            special_tokens=OmegaConf.to_container(cfg.special_tokens) if cfg.special_tokens else None,
+            vocab_file=cfg.vocab_file,
+            vocab_size=cfg.vocab_size,
+            do_lower_case=cfg.do_lower_case,
+        )
+        self.tokenizer = tokenizer
+
```

Comment:
should we define a NLP model and define all such general method there? like an ASR/ctc model as a base class?

Reply:
yes, in separate PR